app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Api::V1
+  class OrdersController < ApplicationController
+    before_action :ensure_params_exist, :load_products, only: [:create]
+    before_action :load_order, only: [:show]
+    skip_before_action :verify_authenticity_token
+
+    def create
+      @orders = Order.new order_params
```

Comment:
tạo mới 1 order thì để tên biến số ít thôi em
```suggestion
      @order = Order.new order_params
```

Reply:
vâng ạ